### PR TITLE
Add SPM support in the search for queries 

### PR DESCRIPTION
**Pull request by orta**
Re: #103 

I think this is the way to go, right @floere - a category which can be used to filter the results? 

Note: you might want to review with `?w=1` on the URL, my editor ate a lot of whitespace.
Note: to get `bundle install` to work on el cap, I had to make this, uncommitted change:

``` diff
diff --git a/Gemfile.lock b/Gemfile.lock
index 1d13ec2..8523e30 100644
--- a/Gemfile.lock
+++ b/Gemfile.lock
@@ -51,7 +51,7 @@ GEM
     foreman (0.75.0)
       dotenv (~> 0.11.1)
       thor (~> 0.19.1)
-    google_hash (0.8.8)
+    google_hash (0.9.0)
       sane (~> 0)
     hashie (3.3.2)
     i18n (0.7.0)
@@ -165,4 +165,4 @@ DEPENDENCIES
   yajl-ruby

 BUNDLED WITH
-   1.10.5
+   1.11.2
```

### CODE REVIEW

**Comment by czechboy0** [comment]
:blue_heart: :blue_heart: :blue_heart:

**Comment by segiddins** [comment]
`/home/travis/build/CocoaPods/search.cocoapods.org/vendor/bundle/ruby/2.2.0/gems/flounder-1.0.1/lib/flounder/connection.rb:22:in `exec': ERROR:  column cocoadocs_pod_metrics.spm_support does not exist (PG::UndefinedColumn)`

**Comment by orta** [comment]
@floere poke - is this the right direction?

**Comment by floere** [comment]
@orta Thanks for the poke! I can check tonight 😊

On Mi., 16. März 2016 at 18:06 Orta

<
mailto:Orta <notifications@github.com>
> wrote:

https://github.com/floere
poke - is this the right direction?

**Comment by floere** [comment]
@orta It is – but it depends what values are in `spm_support`…? Also, I'd not add too many qualifiers. `:spm` alone is enough, I'd say.

Also needed (for completeness' sake):

* [ ] DB migration in Humus.
* [ ] UI stuff in cocoapods.org.

**Comment by orta** [comment]
`spm_support` is a bool in the db ( that has existed for a few months ) - I guess maybe the fixtures need updating?

**Comment by czechboy0** [comment]
I'm really like to see this option added to the search API. Can I help out to accelerate this? 😊

**Comment by orta** [comment]
I've brought this back up to master, and I created the db dump, but I'm currently blocked on being able to upload it into the S3 bucket

File: lib/models/pod.rb [hunk 1]
```diff
@@ -74,13 +74,13 @@ def self.find
         ) AS popularity
         EXPR
         *Domain.github_metrics.fields(:forks, :stargazers, :contributors, :subscribers, :language),
-        *Domain.cocoadocs_pod_metrics.fields(:id, :dominant_language, :quality_estimate)
+        *Domain.cocoadocs_pod_metrics.fields(:id, :dominant_language, :quality_estimate,  :spm_support)
```
Comment (dnkoutso) [inline lib/models/pod.rb]: nit: extra space.

**Comment by dnkoutso** [comment]
This LGTM. Not sure if you want others to review who are more familiar.

**Review by endocrimes — APPROVED** [review]

**Comment by floere** [comment]
👍

**Pull request merged**